rasa/nlu/test.py

```diff
@@ -1011,6 +1030,7 @@ def run_evaluation(
     :param confmat: path to file that will show the confusion matrix
     :param histogram: path fo file that will show a histogram
     :param component_builder: component builder
+    :param no_plot: if true confusion matrix and histogram will not be rendered
```

Comment:
We should reformat this docstring to use the Google format in the future! (not in this PR though)

Reply:
sounds good, is it already part of our code style convention?

Reply:
yes it is https://www.notion.so/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e

Reply:
would be great if we can find a way to test this, e.g. with flake8

regarding notion: should probably also live in this repo or the docs as it is also relevant for contributors and good to point them to this document in PR reviews